src/legacy/useMolecule.tsx

```diff
@@ -0,0 +1,11 @@
+import { Molecule } from "./molecule";
+import { MoleculeScopeOptions } from "./MoleculeScopeOptions";
+
+export function useMolecule<T>(
+  m: Molecule<T>,
+  options?: MoleculeScopeOptions
+): T {
+  // const scopes = useScopes(options);
+  // const store = useStore();
+  return {} as T;
```

Comment:
not sure if this is supposed to be returning something else

Reply:
It's only used for type tests, so nothing in `legacy` is run at runtime